quickstart/101-attestation-provider/providers.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+
```

Comment:
Can we remove this blank line please since it'll cause some warning on some community linters?